CITATION.cff

```diff
@@ -21,6 +21,10 @@ authors:
     family-names: Way
     orcid: 'https://orcid.org/0000-0002-0503-9348'
 repository-code: 'https://github.com/WayScience/coSMicQC'
+identifiers:
+  - description: Software DOI
+    type: doi
+    value: "10.5281/zenodo.14797008"
```

Comment:
I can't remember if you already answered this in our discussion, but will this automatically update when we make new releases?

Reply:
Thanks - yes, this will automatically redirect to the latest. For example, navigating to https://doi.org/10.5281/zenodo.14797008 will redirect to https://doi.org/10.5281/zenodo.14797009 currently.